data/json/items/comestibles/carnivore.json

```diff
@@ -86,7 +86,7 @@
     "volume": 1,
     "price": 500,
     "spoils_in": "1 day",
-    "calories": 250,
+    "calories": 449,
```

Comment:
Should the calories of fish fillets be increased?  They were previously 80% of the calories of meat and I suspect we should keep that proportion.

Reply:
I can lookup a type of fish and get the correct value for it instead of trying to base it off just meat but I would need a type of fish to base it from.

Reply:
Take an average of salmon, trout, carp, pike, and bass, I'd guess.

Reply:
Got average based on the table below and adjusted the calorie amounts. 
Salmon 261
Trout 237
Carp 253 
Pike 176
Bass 227 
--------
Average 230.8